src/pages/Home/components/Info.styles.js

```diff
@@ -0,0 +1,31 @@
+import styled from 'styled-components';
+
+export const InfoSectionContainer = styled.section`
+  width: 100%;
+  padding: 134px 352px 0 352px;
+`;
+
+export const Heading = styled.h2`
+  font-size: ${({ theme }) => theme.fontSizes.large};
+`;
+
+export const List = styled.ul`
+  list-style: none;
+  margin-top: 13px;
+  margin-bottom: 105px;
+`;
+
+export const ListItem = styled.li`
+  line-height: 27px;
+`;
+
+export const Text = styled.p`
+  margin-top: 15px;
+  line-height: 27px;
```

Comment:
you can set `letter-spacing: 0.03px` here. The text gets blurry at the end of a line atm

![image](https://user-images.githubusercontent.com/9722792/78223609-7ba5d180-74c7-11ea-80f5-6dc2e030f041.png)